build/squire-raw.js

```diff
@@ -1572,8 +1572,11 @@ var firstOrLastLine = function(self){
     var so    = range.startOffset
   }
   var parentBlock = getStartBlockOfRange(range)
+  if(parentBlock.nodeName == "LI"){
+    parentBlock = parentBlock.parentElement
+  }
   var numLines    = numberOfLines(root)
```

Comment:
I'd have preferred to include this condition in getStartBlockOfRange, or to modify our inline/block element regex, but doing so introduced side effects elsewhere in the code. The trouble is, lists are a bit of an unusual case, in that ```li``` are block elements that are nested inside ol/ul tags, which are also block elements, which is unusual in squire. I say best to keep treating lis as block elements (which they are) and handle this specific line number issue explicitly.